src/conf.c

```diff
@@ -447,6 +447,7 @@ int conf_loadConfig ( const char* file )
       conf_loadBool("devmode",conf.devmode);
       conf_loadBool("devautosave",conf.devautosave);
       conf_loadBool("conf_nosave",conf.nosave);
+      conf_loadBool("firstran", conf.firstran);
```

Comment:
Why firstran and not firstrun? ;)

Reply:
Because "firstrun" wouldn't be accurate, it would suggest that this is the first time the game is running. What it actually records is that the game *already* ran for the first time in the past.

🕵️